backend/src/backend/main.py

```diff
@@ -77,17 +77,20 @@ async def add_book(book_payload: BookAddSchema):
     )
 
     for metadata_name, value in book_payload.metadata.items():
-        kind = (
-            KIND_TEXT
-            if not metadata_name.startswith("Illustration_")
-            else KIND_ILLUSTRATION
-        )
-        await BookMetadata.objects.create(
-            book=book.id,
-            name=metadata_name,
-            value=value,
-            kind=kind,
-        )
+        if metadata_name.startswith("Illustration_"):
+            await BookMetadata.objects.create(
+                book=book.id,
+                name=metadata_name,
+                bin_value=value,
```

Comment:
Shouldn't that be base64 decoded ?

Reply:
The string is base64 encoded ; look at the tests for an example of what we want to do